.travis.yml

```diff
@@ -59,7 +60,7 @@ before_install:
   - composer config repositories.social git https://github.com/goalgorilla/open_social.git
   # Remove roave/security-advisories depdendency as it conflicts with drupal/core:~8.9.9 requirements in our update branch. See: https://github.com/Roave/SecurityAdvisories/commit/5bd8c4f672d819d073ca8de1922fdb6898d0df3a
   - if [ "$TEST_SUITE" = "install_update" ]; then composer remove roave/security-advisories --no-update; fi
-  # 8.x-8.x-composer-update-to-10-branch has some composer breaking changes for scaffolding, we couldnt release but helps us with testing our update path.
+  # 8.x-8.x-composer-update-to-11-branch has some composer breaking changes for scaffolding, we couldn't release but helps us with testing our update path.
```

Comment:
The update path is broken because Travis is now running PHP 8.0.

Our docker container will run PHP 7.4 which will be based on drupal_social:2.0.0.

So, when we do a composer install / require on host (Travis), that installs dependencies using PHP 8.0 and because PHP 7.4 is running inside docker, it will complain whenever you try to install Open Social.

There are the following solutions:

1.  This is dirty way. Checkout to new branch from 2.0.0 tag of Drupal Social, add the following to composer.json and use the new branch in place of drupal_social:2.0.0
```
"config": {
        "platform": {
            "php": "7.4"
        }
    },
```
2. Or, add PHP 7.4 on travis and configure the matrix to run only upgrade path.
3. Or, ditch the upgrade path testing. 😓 w